src/MobileDevice/widget/MobileDevice.xml

```diff
@@ -44,6 +44,14 @@
                 <attributeType name="String"/>
             </attributeTypes>
         </property>
+        <property key="deviceVersionAttribute" type="attribute" required="false">
+            <caption>Device vesrion</caption>
+            <category>Data source</category>
+            <description>Operating system version of the device</description>
+            <attributeTypes>
+                <attributeType name="String"/>
+            </attributeTypes>
+        </property>
         <property key="onDeviceReadyAction" type="enumeration" defaultValue="doNothing">
```

Comment:
Add description.
`Action to execute when the device is ready`

Reply:
Added

Reply:
Done